src/test/shared/filesystem.test.ts

```diff
@@ -6,271 +6,27 @@
-            await writeFile(filePath, '', 'utf8')
-            const actual = await stat(filePath)
-
-            assert.ok(actual)
-            assert.strictEqual(actual.isFile(), true)
-            assert.strictEqual(actual.isDirectory(), false)
-        })
-
-        it('gets metadata for non-empty file', async () => {
-            await writeFile(filePath, 'Hello, World!', 'utf8')
-            const actual = await stat(filePath)
-
-            assert.ok(actual)
-            assert.strictEqual(actual.isFile(), true)
-            assert.strictEqual(actual.isDirectory(), false)
-        })
-
-        it('rejects if file does not exist', async () => {
-            let error: Error | string | undefined
-
-            try {
-                await stat(filePath)
-            } catch (err) {
-                error = err  as Error | string | undefined
-            } finally {
-                assert.ok(error)
-            }
-        })
-    })
-
-    describe('writeFileAsync', () => {
-        it('writes text data to file', async () => {
-            await writeFile(filePath, 'Hello, World!', 'utf8')
-            const actual = await readFile(filePath, 'utf8')
-
-            assert.ok(actual)
-            assert.strictEqual(actual, 'Hello, World!')
-        })
-
-        it('writes binary data to file', async () => {
-            await writeFile(filePath, 'Hello, World!', 'binary')
-            // tslint:disable-next-line:no-null-keyword
-            const actual = await readFile(filePath)
-
-            assert.ok(actual)
-            assert.strictEqual(actual instanceof Buffer, true)
-            assert.strictEqual((actual as Buffer).toString('binary'), 'Hello, World!')
-        })
-
-        it('overwrites existing file', async () => {
-            await writeFile(filePath, 'Hello, Tom!', 'utf8')
-            await writeFile(filePath, 'Hello, Jane!', 'utf8')
-            const actual = await readFile(filePath, 'utf8')
-
-            assert.ok(actual)
-            assert.strictEqual(actual, 'Hello, Jane!')
-        })
-
-        it('creates file if it doesn\'t already exist', async () => {
-            await writeFile(filePath, 'Hello, World!', 'utf8')
-            const actual = await readFile(filePath, 'utf8')
-
-            assert.ok(actual)
-            assert.strictEqual(actual, 'Hello, World!')
-        })
-
-        it('rejects if directory does not exist', async () => {
-            const wrongFolder = path.join(path.dirname(tempFolder), path.basename(tempFolder) + 'WRONG')
-            const wrongFilePath = path.join(wrongFolder, filename)
-            let error: Error | string | undefined
-
-            try {
-                await writeFile(path.join(tempFolder, wrongFilePath), 'Hello, World!', 'utf8')
-            } catch (err) {
-                error = err  as Error | string | undefined
-            } finally {
-                assert.ok(error)
-            }
+describe('filesystem', () => {
```

Comment:
I think it's fine to remove these tests altogether.

Reply:
The tests server their purpose of ensuring that `filesystem` returns promisified versions of specific functions. Removing them would be a step backwards in real-world coverage.